### PR TITLE
Add a contributed models section with the Portuguese model.

### PR DESCRIPTION
**Pull request by schmmd**

### CODE REVIEW

**Comment by schmmd** [comment]
@matt-peters, here's a proposed change to host contributed ELMo models.  I would like to put more effort into the design once we have more than one contributed model ;-)

File: elmo.html [hunk 1]
```diff
@@ -102,6 +102,27 @@ <h1>Pre-trained ELMo Models</h1>
...
+      <tr style="border-bottom: 2px solid black;">
+        <td style="border-right: 1px solid black;"><b>Model</b></td>
+        <td><b>Link(Weights/Options File)</b></td>
+        <td style="border-right: 2px solid black;">&nbsp;</td>
+        <td><b>Contributor</td>
+      </tr>
+      <tr>
+        <td style="border-right: 1px solid black;">Portuguese</td>
+        <td><a href="https://s3-us-west-2.amazonaws.com/allennlp/models/elmo/contributed/pt/elmo_pt_weights.hdf5">weights<a/></td>
+        <td style="border-right: 2px solid black;"><a href="https://s3-us-west-2.amazonaws.com/allennlp/models/elmo/contributed/pt/elmo_pt_options.txt">options<a/></td>
```
Comment (matt-peters) [inline elmo.html]: This is a `json` file, please rename it to have that extension.

Comment (matt-peters) [inline elmo.html]: This is a `json` file, please rename it to have that extension.

File: elmo.html [hunk 2]
```diff
@@ -102,6 +102,27 @@ <h1>Pre-trained ELMo Models</h1>
...
+        <td>Federal University of Goiás (UFG). Pedro Vitor Quinta de Castro, Anderson da Silva
+          Soares, Nádia Félix Felipe da Silva, Rafael Teixeira Sousa, Ayrton Denner da Silva Amaral.
+          Sponsered by Data-H and Aviso Urgente.
```
Comment (pvcastro) [inline elmo.html]: Can you also add "Americas Health Labs" as sponsor?

**Pull request merged**